### DISCUSSION TITLE
When debugging using VSCode, it takes time for the browser to complete loading.

### DISCUSSION THREAD

**Discussion by tyokoyamax** [vercel/next.js#66733] [Help]
### Summary

* OS : Ubuntu24.04 LTS (Windows10 WSL)
* node : v20.14.0
* npm : 10.7.0
* next : 14.0.2
* react : 18.2.0
* vscode : 1.90.0

I am planning to develop using Nextjs and VSCode.

The project was created as in the tutorial.
https://nextjs.org/learn/dashboard-app/getting-started

While checking the functionality, I noticed that the browser is taking a long time to load.
This will only occur if a breakpoint is set in vscode. It will not occur if “npm run dev” or no breakpoints are set.

What are the causes and solutions?

https://github.com/vercel/next.js/assets/33122583/f74701bd-6a47-4825-a15c-16f458b78937

![2024-06-11 131401](https://github.com/vercel/next.js/assets/33122583/1c4eea14-82ed-4b88-b9d8-07c096d591b4)

launch.json

```json
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Next.js: debug server-side",
            "type": "node-terminal",
            "request": "launch",
            "command": "npm run dev"
        },
        {
            "name": "Next.js: debug client-side",
            "type": "chrome",
            "request": "launch",
            "url": "http://localhost:3000"
        },
        {
            "name": "Next.js: debug full stack",
            "type": "node-terminal",
            "request": "launch",
            "command": "npm run dev",
            "serverReadyAction": {
                "pattern": "- Local:.+(https?://.+)",
                "uriFormat": "%s",
                "action": "debugWithChrome"
            }
        }
    ]
}
```

### Additional information

```js
may be relevant
https://github.com/vercel/next.js/issues/62008
```

**Comment by TimurKr** [comment]
Any news on this? Experiencing the same thing an it gets very annoying after a while. I haven't been able to figure out what is causing this.
